### PR TITLE
Fix publish related errors.

### PR DESCRIPTION
**Pull request by chitang**

### CODE REVIEW

File: src/main/java/com/socrata/api/Soda2Producer.java [hunk 1]
```diff
@@ -193,7 +193,7 @@ public ClientResponse issueRequest() throws LongRunningQueryException, SodaError
      *     <li>If an object already exists with this value, it will update it.</li>
      *     <li>If the object has a :deleted=true value, the object will be deleted.</li>
      * </ol>
-     *   <br/>
+     *   <br>
```
Comment (aerust) [inline src/main/java/com/socrata/api/Soda2Producer.java]: why this change?

**Comment by chitang** [comment]
javadoc in java 8 complains about self-closing element not allowed

**Comment by aerust** [comment]
Okay, cool. Maybe make the commit message convey "fix java doc" more? Other than that, looks good to me. :sheep:
